src/h5web/visualizations/shared/models.ts

```diff
@@ -45,7 +38,7 @@ export type AxisScale =
   | ScaleSymLog<number, number>;
 
 export interface AxisInfo {
-  isIndexAxis: boolean;
+  onlyIntegers?: boolean;
```

Comment:
`isIndexAxis` is only used to check whether ticks should only display integers so I think this renaming made sense.

Reply:
Hehe, random thought, but in the end, showing only integers on the heatmap axes is equivalent to not showing interpolated ticks when passing an array of abscissa or ordinate values.

Reply:
Well spotted ! We should indeed keep this in mind.